src/osp/Resource/AssetImporter.cpp

```diff
@@ -55,38 +56,41 @@ using Magnum::GL::Mesh;
 namespace osp
 {
 
-void osp::AssetImporter::load_sturdy_file(std::string const& filepath, Package& pkg)
+void osp::AssetImporter::load_sturdy_file(std::string_view filepath, Package& pkg)
 {
     PluginManager pluginManager;
     TinyGltfImporter gltfImporter{pluginManager};
 
     // Open .sturdy.gltf file
-    gltfImporter.openFile(filepath);
+    gltfImporter.openFile(std::string{filepath});
```

Comment:
gltifImporter.openFile doesn't support std::string_view?

Does it take an std::filesystem::path or something?

Reply:
I tried leaving it that way, but it said:
```
osp-magnum/src/osp/Resource/AssetImporter.cpp:65:35: error: no matching function for call to ‘Magnum::Trade::TinyGltfImporter::openFile(std::string_view&)’
     gltfImporter.openFile(filepath);
```

It takes a `const std::string&`. Should it work? If not, it's a small price to pay for avoiding the many allocations from `substr`ing the `std::string` as before.

Reply:
How it is is the best you're going to be able to do unless we can convince the Magnum folks to support std::string_view in their APIs.

So this is fine.